flask_resty/pagination.py

```diff
@@ -63,6 +64,18 @@ def parse_limit(self, limit):
 
         return limit
 
+    def spec_declaration(self, path, spec, **kwargs):
+        path['get'].add_parameter(name='limit',
+                                  type='int',
+                                  description='pagination limit')
+        path['get'].add_property_to_response(
+            prop_name='meta',
+            type='object',
+            properties={
```

Comment:
This isn't a merge operation, is it? Seems a little weird then – `meta` might contain metadata from multiple other extensions.